src/connect/ConnectWorkspaceFolder.ts

```diff
@@ -165,8 +163,8 @@ export class ConnectWorkspaceFolder extends WorkspaceFolderBase {
             return false;
         }
 
-        const bridgeClient: BridgeClient = await this._binariesUtility.tryGetBridgeAsync();
-        const kubectlClient: KubectlClient = await this._binariesUtility.tryGetKubectlAsync();
+        var bridgeClient: BridgeClient;
+        var kubectlClient: KubectlClient;
```

Comment:
💡 empty object for `bridgeClient` and `kubectl` they both will return false under line 168.